### PR TITLE
Card specific cta

### PR DESCRIPTION
**Pull request by KarafiziArtur**
https://github.com/local-motion/product/issues/357

### CODE REVIEW

File: src/views/Workspace/Cards/CollectOpinionsCard.jsx [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
+    componentWillUnmount() {
+        this.props.unsetCta();
+    }
+
+    setCta() {
+        const { setCta, playground, user, history } = this.props;
+
+        setCta({
+            ctaAction: () => history.push(`${getWorkspaceStartLink(playground)}/team`),
+            ctaText: 'Chat with teammembers',
```
Comment (maxvisser) [inline src/views/Workspace/Cards/CollectOpinionsCard.jsx]: 'Chat with teammembers' translates into 'Chat met teamleden'

Comment (maxvisser) [reply src/views/Workspace/Cards/CollectOpinionsCard.jsx]: clicking this button should also open up community category;

Comment (KarafiziArtur) [reply src/views/Workspace/Cards/CollectOpinionsCard.jsx]: Good catch. Thanks! Fixed.

File: src/views/Workspace/Cards/CollectOpinionsCard.jsx [hunk 2]
```diff
@@ -1,8 +1,39 @@
...
+
+    componentWillUnmount() {
+        this.props.unsetCta();
+    }
+
+    setCta() {
+        const { setCta, playground, user, history } = this.props;
+
+        setCta({
+            ctaAction: () => history.push(`${getWorkspaceStartLink(playground)}/team`),
```
[the remark this answers is not in the document]
Comment (dimitriirtimid) [reply src/views/Workspace/Cards/CollectOpinionsCard.jsx]: decided on yes -> link to /chat

**Pull request closed without merging**